app/scripts/models/date.coffee

```diff
@@ -0,0 +1,38 @@
+root = exports ? this
+
+root.Date = class Date
```

Comment:
Renamed to GameDate. Phase is teetering on being overloaded. I propose leaving it as is in the pull, we can revisit later.